api_tests/preprints/views/test_preprint_list.py

```diff
@@ -377,6 +403,22 @@ def test_create_preprint_from_project_unpublished_does_not_hit_update(self, mock
         res = self.app.post_json_api(self.url, private_project_payload, auth=self.user.auth)
         assert not mock_on_preprint_updated.called
 
+    @mock.patch('website.preprints.tasks.get_and_set_preprint_identifiers.si')
```

Comment:
Not specific to this PR, but my general comment with mocks is to double-check to make sure that it is actually testing a thing. Since you're getting a 409 below, it probably is, but these are tricky and have sat around doing nothing before, so just in case.

Reply:
I copied it from the other tests, looks like it's not actually testing a thing.